clients/src/test/java/org/apache/kafka/common/utils/ChunkedBytesStreamTest.java

```diff
@@ -36,6 +31,12 @@
 import java.util.Random;
 import java.util.stream.Stream;
 
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.spy;
+
```

Comment:
Please avoid changing the order of imports. It adds to unnecessary code review visual clutter.